atd-vze/src/views/Crashes/Recommendations/Recommendations.js

```diff
@@ -3,7 +3,7 @@ import { useQuery, useMutation } from "@apollo/react-hooks";
 import { Card, CardHeader, CardBody, CardFooter } from "reactstrap";
```

Comment:
I am for the first time noticing these empty `<CardFooter/>` components in the notes and recs, and also all the data table cards 👀 

I'm going to make a snackoo to clean these up bc they are clutter imo 🧹